pkg/controller/controller_instance.go

```diff
@@ -496,6 +527,8 @@ func (c *controller) reconcileServiceInstanceUpdate(instance *v1beta1.ServiceIns
 func (c *controller) reconcileServiceInstanceDelete(instance *v1beta1.ServiceInstance) error {
 	// nothing to do...
 	if instance.DeletionTimestamp == nil && !instance.Status.OrphanMitigationInProgress {
+		// TODO nilebox: shouldn't we throw an error instead?
```

Comment:
What do people think?

Reply:
Currently it is success, yes, or rather "nothing to do".
But should never end up in this condition being true IMO.